app/src/main/java/org/simple/clinic/home/overdue/search/OverdueSearchUiRenderer.kt

```diff
@@ -10,6 +10,8 @@ class OverdueSearchUiRenderer(
     if (!model.hasSearchQuery) {
       ui.showSearchHistory(model.overdueSearchHistory.orEmpty())
       ui.hideSearchResults()
+    } else if (model.hasOverdueSearchResults) {
+      ui.showOverdueSearchResults(model.overdueSearchResults!!)
```

Comment:
When there is search query and search results, we also have to hide the search history. Should we also handle that here? or will you raise a different PR?